lib/Auth/Utils.php

```diff
@@ -76,7 +76,7 @@ public static function authenticatePI(array &$state, array $formParams): ?PIResp
                     // If the authentication has been confirmed on the phone, the authentication has to be finalized with a
                     // call to /validate/check with an empty pass
                     // https://privacyidea.readthedocs.io/en/latest/tokens/authentication_modes.html#outofband-mode
-                    $response = $pi->validateCheck($username, "", $transactionID);
+                    $response = $pi->validateCheck($username, "", $transactionID, $headersToForward);
```

Comment:
rename $headersToForward to $headers

Reply:
fixed [f2bb187](https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/198/commits/f2bb187f584e1a7a8c5c5fec38dee8d19a3e6b78)